ibc_tests/transfer_test.go

```diff
@@ -59,15 +59,19 @@ type TransferTestSuite struct {
 
 func (suite *TransferTestSuite) SetupTest() {
 	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 3)
-	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(0))
-	suite.chainB = suite.coordinator.GetChain(ibctesting.GetChainID(1))
-	suite.chainC = suite.coordinator.GetChain(ibctesting.GetChainID(2))
+	// they changed from 0-indexed to 1-indexed, I don't understand why
+	// all I know is that MY CODE IS BROKEN THANKS TO THAT
```

Comment:
seems to relate to this https://github.com/cosmos/ibc-go/pull/774#discussion_r788721844

Reply:
It reads like "we serve big chains, you small chain fxxk off"